internal_transfers/actions/src/simulate/tenderly.ts

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+import { TxSimulator } from "./interface";
+
+export class TenderlySimulator extends TxSimulator {
+  BASE_URL = "https://api.tenderly.co/api";
+  user: string;
+  project: string;
+  accessKey: string;
+
+  constructor(user: string, project: string, accessKey: string) {
+    super();
+    this.user = user;
+    this.project = project;
+    this.accessKey = accessKey;
+  }
+  async simulate(
+    contractAddress: string,
+    sender: string,
+    callData: string,
+    save: boolean,
+    blockNumber?: number
+  ): Promise<string> {
```

Comment:
Not so sure about the type here... is `response.data` actually a string? cc @alfetopito?

Reply:
I changed it to any....

Reply:
According to [axios' docs](https://axios-http.com/docs/res_schema), it'll be whatever is the response you get, I assume a JSON already, which you could then map it to the type

Reply:
uhhh.... map it to what type? Is there a json "type"?

Reply:
I don't want to map to the expected type within this method becuase I want to keep the response parsing as a separate method that can be unit tested without having to make a request. So I will just leave this and return the raw response here.